<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaFluxoComportamental</class>
 <widget class="QWidget" name="telaFluxoComportamental">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>410</y>
         <width>891</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Tutor</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>171</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Step 1.3</string>
       </property>
       <widget class="QPushButton" name="pbSaveFile">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit File</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>141</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>731</width>
         <height>291</height>
        </rect>
       </property>
       <property name="title">
        <string>Chose Category</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>709</width>
             <height>256</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QWidget" name="widCatego" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>161</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Step 1.1</string>
       </property>
       <widget class="QComboBox" name="comboBoX">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Microsoft RLE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Microsoft Video1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Intel IYUV codec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cinepack Codec by radius</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lagarith Lossles Codec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>H264</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XVid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ffdshow Video Codec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>H264</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>28</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Codec</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Step 1.0</string>
       </property>
       <widget class="QPushButton" name="pbAnaliseFile">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Open TCC file</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>161</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Step 1.2</string>
       </property>
       <widget class="QPushButton" name="pbOpenTCCM">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Open TCCM file</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>691</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
